### DISCUSSION TITLE
Support Logitech bluetooth

### DISCUSSION THREAD

**Discussion by isidoro-80** [darthcloud/BlueRetro#855] [Q&A]
Hi everyone, I have a very nice logitech bluetooth gamepad for ps2 model precision cordless 2.4 ghz frequency. I lost the key that fits into the gamepad port that allows you to connect the gamepad to the PS2.  Without this key I can't make it work.  each key is unique for each gamepad.  I can't use another key on an identical ganepad of the same brand.  with blueretro could I reuse this gamepad and connect it to the ps2 and other consoles?

**Comment by darthcloud** [comment]
It's not Bluetooth, it's proprietary 2.4GHz.

So not possible.

**Comment by isidoro-80** [comment]
Thanks
